root/static/styles/forms.less

```diff
@@ -530,6 +530,7 @@ input.icon, div.icon.img, span.icon.img, button.icon, a.icon {
     background-color: transparent;
     vertical-align: middle;
     cursor: pointer;
+    &:disabled { cursor: default; }
```

Comment:
It might be slightly better to move this to a separate block for only `input.icon, button.icon` so that CSS isn't unnecessarily generated for elements that don't support a `:disabled` pseudo-class (even though I believe it's a no-op). Here's the current compiled output:

```CSS
input.icon:disabled,
div.icon.img:disabled,
span.icon.img:disabled,
button.icon:disabled,
a.icon:disabled {
  cursor: default;
}
```

Reply:
Sure, makes sense!

Reply:
(It'll be nice when https://caniuse.com/css-nesting can just be used! Chrome and Safari didn't get it until late last year, though.)